boxes.json

```diff
@@ -4,55 +4,79 @@
 		"topRight": "┐",
 		"bottomRight": "┘",
 		"bottomLeft": "└",
-		"vertical": "│",
-		"horizontal": "─"
+		"verticalLeft": "│",
```

Comment:
What do you think of just making these `left`, `right, `top`, `bottom`?

Reply:
Good thought. Would make it more consistent with "bottomRight" and would fit better to the word box (left side etc.)

Reply:
Ok. Let's do that then.